package.json

```diff
@@ -38,13 +39,16 @@
     "gulp-uglify": "^0.3.1",
     "gulp-useref": "^0.5.0",
     "gulp-util": "^3.0.0",
-    "gulp-protractor": "^0.0.11",
     "ink-docstrap": "^0.4.12",
+    "jasmine-core": "^2.1.3",
```

Comment:
Oops!  Totally sorry I didn't see this before...  we've actually been using mocha for our protractor tests in stellar-client (and indeed, across all of our JS projects).  Sorry you wasted your time on it.

If you would, could you switch over to mocha/sinon/chai?  If you don't have much experience with them, I'm happy to help.